samples/MicrophoneStream.js

```diff
@@ -0,0 +1,129 @@
+/**
```

Comment:
Please remove Microphone Streaming

Let me know if I'm mistaken and it should be included. AFAIK this PR is for How-to snippets only.